### PR TITLE
Overdue wristbound resprite : Death of Retro-futurism

### PR DESCRIPTION
**Pull request by alsoandanswer**
dreammaker_09cTYlIgFC

I'm also changing the colors back, because after reading [the feedback thread](https://forums.aurorastation.org/topic/14816-adjust-the-wristbound-computer-sprites) more closely, a lot of people were fine with the colors, just not the onmobs.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by TheDocOct** [comment]
Feedback thread, please

**Comment by alsoandanswer** [comment]
i feel the feedback thread above was enough, but if a maintainer asks, sure

File: code/_helpers/global_lists.dm [hunk 1]
```diff
@@ -63,7 +63,7 @@ var/global/list/backbagstyles = list("Job-specific", "Grey")
 var/global/list/exclude_jobs = list(/datum/job/ai,/datum/job/cyborg, /datum/job/merchant)
 
 //PDA choice
-var/global/list/pdalist = list("Nothing", "Standard PDA", "Classic PDA", "Rugged PDA", "Slate PDA", "Smart PDA", "Tablet", "Wristbound")
```
Comment (Greenjoe12345) [inline code/_helpers/global_lists.dm:66]: is removing the tablet intended?

Comment (furrycactus) [reply code/_helpers/global_lists.dm:66]: Is removing _any_ of this intended?

Comment (alsoandanswer) [reply code/_helpers/global_lists.dm:66]: my bad must be a bad merge

**Comment by Geevies** [comment]
i agree with making a feedback thread

**Comment by NonQueueingMatt** [comment]
Think this should go up to the forums. I would like to avoid a fiasco like last time.

**Comment by alsoandanswer** [comment]
https://forums.aurorastation.org/topic/15253-wristbounds-v2/

**Review by skull132 — APPROVED** [review]

File: code/modules/modular_computers/computers/subtypes/preset_pda.dm [hunk 2]
```diff
@@ -198,4 +198,4 @@
 	icon_add = "transp"
 
 /obj/item/modular_computer/handheld/pda/civilian/merchant
-	hidden = TRUE
\ No newline at end of file
+	hidden = TRUE
```
Comment (JohnWildkins) [inline code/modules/modular_computers/computers/subtypes/preset_pda.dm:201]: accidental newline save here

File: code/datums/outfits/outfit.dm [hunk 3]
```diff
@@ -222,14 +222,19 @@
 	if(pda && !visualsOnly)
 		var/obj/item/I = new pda(H)
 		switch(H.pda_choice)
+			I.desc_fluff = "For its many years of service, this model has held a virtual monopoly on the PDA market. The secret? A lapel pin affixed to the back."
```
Comment (JohnWildkins) [inline code/datums/outfits/outfit.dm:225]: this needs to go outside the switch or in its own case, right now this doesn't compile

**Review by JohnWildkins — CHANGES_REQUESTED** [review]
needs a couple minor fixes, and additionally: loadout wristbounds do not get icons set properly. i don't think this is a bug with this PR necessarily, but it should be a simple fix. hit me up on discord about it

**Comment by NonQueueingMatt** [comment]
Inactive author.

**Pull request closed without merging**